src/es/components/molecules/carouselTwo/CarouselTwo.js

```diff
@@ -344,7 +370,7 @@ export default class CarouselTwo extends Mutation() {
         height: var(--nav-height, 1em);
         width: var(--nav-width, 1em);
       }
-      :host(.has-default-nav) > nav > *, :host > nav > * {
+      :host(.has-default-nav) > nav > *, :host > nav > *, :host(.has-default-nav) > div > nav > *, :host > div > nav > * {
         transition: all .3s ease-out !important;
```

Comment:
lost transition

Reply:
@Nilsilino ich würde als Nacharbeit entweder für alle MDX Variablen einen Fallback definieren, so dass die Vorschau auch schön im Toolbox Repo aussieht oder das Carousel in der Klubschule extenden und nur dort diese zwei CSS Themes 3-column und Image im Switch-Case laden.